packages/neo-one-cli/src/cmd/compile.ts

```diff
@@ -10,13 +10,31 @@ export const builder = (yargsBuilder: typeof yargs) =>
     .string('outDir')
     .describe('outDir', 'output code directory')
     .string('path')
-    .describe('path', 'contract directory');
+    .describe('path', 'contract directory')
+    .boolean('json')
+    .default('json', true)
+    .describe('json', 'output the contract with the json format')
+    .boolean('avm')
+    .describe('avm', 'output the contract with the avm format')
```

Comment:
Same here. Capitalize `avm` to `AVM`. This description might need to mention that it will actually produce the zipped `.avmdbgnfo` file.